decred/decred/dcr/wire/msgverack.py

```diff
@@ -0,0 +1,67 @@
+"""
+Copyright (c) 2020, The Decred developers
+See LICENSE for details.
+
+Based on dcrd MsgVerAck.
+"""
+
+from decred.util.encode import ByteArray
+
+
+CmdVerAck = "verack"
+
+
+class MsgVerAck:
+    """
+    MsgVerAck defines a Decred verack message which is used for a peer to
+    acknowledge a version message (MsgVersion) after it has used the information
+    to negotiate parameters.  It implements the Message interface.
```

Comment:
Message interface

Reply:
I don't quite follow here. Are you suggesting a language change?

Reply:
Message interface is an actual interface{} named Message. https://github.com/decred/dcrd/blob/d86202be3992f3e40d9d7fdd9115933a03b5c196/wire/message.go#L65 Since that doesn't exist in python, I would suggest different wording, or just leave all mentions out.

Reply:
Nevermind. I see what you're getting at from your other review. Will update.